src/components/Arping/Arping.tsx

```diff
@@ -0,0 +1,217 @@
+import { useState, useEffect, useCallback } from "react";
+import { Button, Stack, TextInput, Checkbox } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import InstallationModal from "../InstallationModal/InstallationModal";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
```

Comment:
This should be the loading overlay for pkexec
```suggestion
import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/OverlayAndCancelButton";
```

Reply:
Fixed.